packages/components/loading/src/loading-provider.tsx

```diff
@@ -426,14 +426,14 @@ const PageComponent = memo(
   }: LoadingComponentProps) => {
     const css: CSSUIObject = {
       bg: ["white", "black"],
-      maxW: "md",
+      maxW: "24rem",
       p: "md",
```

Comment:
Please refer to `spaces.ts` for reference!
```suggestion
      p: "1rem",
```